### ISSUE TITLE
Option to customize Enter key behavior

### ISSUE THREAD

**Issue by win-open** [FossifyOrg/Keyboard#66]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Keyboard/issues) or [closed](https://github.com/FossifyOrg/Keyboard/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Keyboard/discussions) or [closed](https://github.com/FossifyOrg/Keyboard/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Keyboard/blob/main/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

I want to add an option in the settings to customize the behavior of the Enter key. Currently, pressing the Enter key sends the message, but I want to be able to choose whether it appends a new line or delivers the message.

### Why do you want this feature?

I want this feature because the current behavior of the Enter key is inconsistent with the typical expectation of inserting a new line in text input fields. This causes inconvenience and frustration when using messenger apps. By adding this option, users can choose how they want the Enter key to behave, improving their overall typing experience.

### Additional information

None.

**Comment by Aga-C** [comment]
The behavior of the Enter key depends on the type of the text field. Not every text field accepts a new line, so it wouldn't make sense to do an option to always enforce it. Also, current behavior is the same as in physical keyboards and other software keyboards.

**Comment by win-open** [comment]
I have tested other keyboards like Simple Keyboard by Raimondas Rimkus, FlorisBoard, Gboard etc. When I have tested in messenger apps like Discord, Signal etc, all keyboards default action on pressing enter key is append a new next line but when it comes to Fossify keyboard it's not the same.

Only when I have tested in note taking apps or website like Pastebin then Fossify board appends a new line on pressing enter key. I guess it have problem dealing with messenger platforms.
